src/pages/public_registration/PublicRegistrationStatusBar.tsx

```diff
@@ -53,50 +69,118 @@ export const PublicRegistrationStatusBar: FC<PublicRegistrationContentProps> = (
     entityDescription: { reference },
     doi,
     doiRequest,
+    publisher,
   } = registration;
+  const registrationIsValid = registrationValidationSchema.isValidSync(registration);
 
   const [messageToCurator, setMessageToCurator] = useState('');
-  const [hasPendingDoiRequest, setHasPendingDoiRequest] = useState(doiRequest?.status === DoiRequestStatus.Requested);
   const [openRequestDoiModal, setOpenRequestDoiModal] = useState(false);
-  const [isLoadingDoiRequest, setIsLoadingDoiRequest] = useState(false);
+  const [isLoading, setIsLoading] = useState(LoadingName.None);
```

Comment:
likte denne - men "loading" er kanskje ikke riktig begrep ?  isWaiting ?

Reply:
tja, skjønner hva du mener, men vi bruker "loading" overalt ellers for å indikere at vi venter på svar fra backend, så syns det er like greit å bare holde seg til det her også